### ISSUE TITLE
handling multi-selection / selection confirmation of action for menus

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#2067]
follow on to https://github.com/w3c/aria/issues/2050

menuitems (of all varieties) are historically expected to close upon a menuitem(checkbox/radio) being chosen.  For the latter types, this can potentially lead to ambiguity in confirmation of action (did the selection of the menuitemcheckbox/radio take effect?  how can one tell if not navigating to the content that may have been changed, or re-opening the menu to find the chosen item to verify that it is now in the checked state?)

Additionally, when a menu contains a variety of menuitemcheckboxes which can each be chosen - and at times someone MAY want to choose multiple items at once - if following the traditional pattern of closing upon selection, someone would have to keep re-opening the menu to choose their next item, and repeat until they were done.

Per the APG Editor Menubar example that @giacomo-petri referenced in todays' call, the 'style/color' menuitems are a good example of containing a variety of menuitems that someone might want to turn on all at once, but to do so one must choose an item, have the menu close, open it again, choose the next, etc.

While this may not be the solution the group goes with, could one solution to this issue be specifically identify a menu or a group within a menu as supporting multi-selection - so while regular menuitems or menuitemradios may perform their action and close the menu immediately, a menu or sub-group of menuitemcheckboes could be explicitlly called out as allowing for multi-selection - and then specific expectations for how one can confirm their selections/close the menu can be made / AT can then pass on these expected behaviors to users via their own hints, as applicable?

Or, _other ideas go here_??

**Comment by spectranaut** [comment]
postpone until shorter agenda

**Comment by scottaohara** [comment]
we discussed this before  going on holiday break.  one main takeaway was how to close a multi-select menu if selecting a menuitem does not close the menu.   esc is generally thought of as a 'cancel and close' action.  would there need to be an explicit close button?

**Comment by spectranaut** [comment]
Minutes from that discussion: https://www.w3.org/2023/12/14-aria-minutes#t05

Should this be discussed again in a meeting, Scott?

**Comment by MarioBatusic** [comment]
If we allow to set a menu or menu group to "multiselectable", we have to offer a "close" menu option. Thou it could be possible to have "spacebar" for selecting without closing the menu and "enter" for selecting and closing, I think it is necessary for mouse and touch devices to have a clear exit point.

**Comment by scottaohara** [comment]
@spectranaut i don't think it needs to necessarily be on an agenda at this point - but it is probably worth anyone who is interested in the topic/trying to sovle for the use case to think up some ideas on what could be done here - cause it's a real-world need that we don't have a good solution for right now.
